ion-schema/src/isl/isl_type.rs

```diff
@@ -44,7 +44,15 @@ impl IslType {
     }
 
     /// Verifies if the [IslType] allows open content or not
-    pub fn open_content(&self) -> bool {
+    pub fn is_open_content_allowed(&self) -> bool {
+        match &self {
+            IslType::Named(named_type) => named_type.is_open_content_allowed(),
+            IslType::Anonymous(anonymous_type) => anonymous_type.is_open_content_allowed(),
+        }
+    }
```

Comment:
Why wouldn't it allow open content? ISL 1.0 always allows open content _in a type definition_. The `content: closed` constraint stops the `fields` constraint from allowing open content in the data that is being validated.

I guess a better question is how/where is this function used?

Reply:
In this context its just used to mark if `content:closed` is used or not?

Reply:
I don't think we should expose it as a `pub fn`, so that users don't get confused the way I was. If a user wants to inspect a type and see if it has `content: closed`, they can check the type for `IslConstraint::ContentClosed`, right?

Reply:
Thats correct. I will change it.